lesson26_hw_catnews/catnews/views.py

```diff
@@ -0,0 +1,26 @@
+from django.shortcuts import render
+from django.views.generic import TemplateView
+from django.views import View
+from catnews.models import News, Comment
+
+# Create your views here.
+
+class HomePageView(View):
+    TITLE = "Home Page"
+    DEFAULT_NEWS_PICTURE = 'media/default.jpg'
+    sort_by = 'pub_date'
```

Comment:
why are you not use caps here? - looks like it is constant.. but you override in the first line into `get` function.. 😕